test/test_torch.py

```diff
@@ -4891,6 +4891,136 @@ def test_diff(self, device, dtype):
             scalar = torch.tensor(2, device=device, dtype=dtype)
             torch.diff(scalar)
 
+    def _test_histogram_numpy(self, t, bins, bin_range, weights, density):
```

Comment:
Looks like the histc tests are in test_reductions.py:

https://github.com/pytorch/pytorch/blob/0a267819664dedb4b419edb0fc07f573c50523fa/test/test_reductions.py#L2264

Is there anything that should be changed there? I suppose we might as well move these tests to be next to them.